Dockerfile

```diff
@@ -4,19 +4,17 @@ MAINTAINER Austin Riba <ariba@lco.global>
 EXPOSE 80
 ENV NODE_ENV production
 WORKDIR /valhalla
-CMD uwsgi --ini /etc/uwsgi.ini
+CMD gunicorn valhalla.wsgi -b 0.0.0.0:80
 
 RUN curl -sL https://deb.nodesource.com/setup_9.x | bash -
 RUN apt-get install -y gfortran nodejs
 
 COPY requirements.txt .
-RUN pip install numpy && pip install uwsgi -r requirements.txt
+RUN pip install numpy && pip install gunicorn -r requirements.txt
```

Comment:
Should we pin specific versions to avoid problems?

Reply:
Done